Topic1/src/com/topic1/ShoppingCart.java

```diff
@@ -0,0 +1,70 @@
+package com.topic1;
+
+import java.util.HashSet;
+//import java.util.HashSet;
+//import java.util.Iterator;
+import java.util.Iterator;
+import java.util.Scanner;
+
+public class ShoppingCart {
+
+	private PayForm pForm;
+	private HashSet<Item> items;
```

Comment:
[Medium] You interfaces (or the most general hierarchy) for declare a variable. In this case, you *Set* instead of HashSet